migrations/2023-06-15-add-tile.js

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env node
+/* eslint no-await-in-loop: off */
+require('dotenv').config()
+const mongo = require('../lib/util/mongo')
+const {calcMetaTilesNumero, calcMetaTilesVoie} = require('../lib/util/tiles')
+
+async function addTilesToNumeros() {
+
+  const numeros = await mongo.db.collection('numeros')
+    .find({_deleted: null})
+    .project({_id: 1, positions: 1})
+    .toArray()
+
+  return Promise.all(
+    numeros
+      .filter(n => n.positions)
+      .map(n => {
+        n = calcMetaTilesNumero(n)
+        return mongo.db.collection('numeros').updateOne({_id : n._id}, {$set: n})
+      })
+  )
+}
+
+async function addTilesToVoies() {
+  const voies = await mongo.db.collection('voies')
+    .find({_deleted: null})
+    .project({_id: 1, typeNumerotation: 1, trace: 1})
+    .toArray()
+
+  return Promise.all(
```

Comment:
Pas fan du Promise.all dans ce cas parce que toutes les updates sont faite en concurrence, pour être sûr qu'il n'y ait pas de problème je partirais plutôt sur une update à la fois. Une truc genre for await ...